### ISSUE TITLE
Problem starting Heaps Hello World on Fedora 36

### ISSUE THREAD

**Issue by linuxfranz** [HaxeFoundation/hashlink#549]
I've compiled hashlink 1.12 without problems on Fedora 36.

When I try to run the Heaps HelloWorld example I get

```
[franz@fedora helloHeaps]$ hl hello.hl 
hl: symbol lookup error: /home/franz/Software/hashlink-1.12/sdl.hdll: undefined symbol: glGetString
```

My `compile.hxml`:

```
-cp src
-lib heaps
-lib hlsdl
-hl hello.hl
-main Main
-debug
```

**Comment by csharpdf** [comment]
I have the exact same error after compiling hashlink with ALANVF's [forked makefile](https://github.com/ALANVF/hashlink/blob/master/Makefile), at least for me I think it's because the makefile was forked about a year ago.

**Comment by tobil4sk** [comment]
@ncannasse It seems the cause of this issue is that `glGetString` is not declared here: https://github.com/HaxeFoundation/hashlink/blob/master/libs/sdl/GLImports.h

Is there a reason for this or was it omitted by accident?

There are many more of these imports that are missing.

**Comment by ncannasse** [comment]
I think we only need the manually defined imports for the ones that are not defined in standard gl headers (OpenGL 2.0 ?)

**Comment by tobil4sk** [comment]
Is the header meant to be in `gl.c`? There are these, but neither of these contain glGetString:
https://github.com/HaxeFoundation/hashlink/blob/master/libs/sdl/gl.c#L32-L33

**Comment by ncannasse** [comment]
I think glu does include <gl\gl.h> which should have glGetString.
Maybe what's missing is not the GL include header  (or else you would get a
warning at C compilation) but the runtime library to be correctly linked
(-Lgl on Linux)

**Comment by tobil4sk** [comment]
Ah yes, that's correct. Adding the linker flag does solve the problem actually. I made a PR to fix the issue in the make file: #555. There is also a PR already to fix this for cmake as well: #516.

**Comment by tobil4sk** [comment]
@ncannasse Thanks! Would you be able to merge the one that fixes this issue for CMake as well: #516 ?

**Comment by linuxfranz** [comment]
master build works fine for me now. Thank you @ncannasse and others for solving this!
